libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -134,6 +134,12 @@ Deprecations and Removals
   `std-allocator-const <https://clang.llvm.org/extra/clang-tidy/checks/portability/std-allocator-const.html>`
   enabled.
 
+- libc++ no longer supports relational comparison for ``std::chrono::weekday``. The relational comparison operators were
+  provided as an undocumented extension. If you were using relational comparison on ``std::chrono::weekday``, compare
+  the results of ``c_encoding()`` or ``iso_encoding()`` instead. The
+  ``_LIBCPP_ENABLE_REMOVED_WEEKDAY_RELATIONAL_OPERATORS`` macro can be defined to temporarily re-enable this extension.
+  This macro will be honored for one release and ignored starting in LLVM 20.
+
```

Comment:
Can you also add an entry in "Upcoming deprecations and removals" for LLVM 20 saying that the macro will be removed? This mostly acts as a reminder for us when we start working on LLVM 20 -- we go through the upcoming deprecations/removals from the previous release and make sure they happen.

Reply:
Done. It seems that `20.rst` should also be updated so I also added an entry inside it.